install/kubernetes/generate-webhook-cert.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/sh
+
+set -e
+
+function usage() {
```

Comment:
This function specification does not work. You can remove the function keyword and just have usage.